styles.css

```diff
@@ -0,0 +1,955 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Montserrat", sans-serif;
+}
+
+a {
+  font-family: 'Roboto', sans-serif;
+  text-decoration: none;
+}
```

Comment:
You still have to address the changes to the fonts required by the previous reviewer. Please do so.